app/src/main/java/org/oppia/app/options/StoryTextSizeFragmentPresenter.kt

```diff
@@ -0,0 +1,131 @@
+package org.oppia.app.options
+
+import android.content.Intent
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.SeekBar
+import androidx.fragment.app.Fragment
+import org.oppia.app.databinding.StoryTextSizeFragmentBinding
+import org.oppia.app.model.StoryTextSize
+import javax.inject.Inject
+
+/** The presenter for [StoryTextSizeFragment]. */
+class StoryTextSizeFragmentPresenter @Inject constructor(private val fragment: Fragment) {
+  private var fontSize: String = getStoryTextSize(StoryTextSize.SMALL_TEXT_SIZE)
```

Comment:
Change the default font to `MEDIUM_TEXT_SIZE`

Reply:
done